### PR TITLE
feat: summary page common layout component and route updates

### PR DESCRIPTION
**Pull request by bbehnke**
Adds summary page layout component for reuse on all summary pages and updates routes back to plural paths.

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -94,7 +95,7 @@ function App() {
     // Cluster installation routing
     return (
       <Routes>
-        <Route path="/" element={<div>TODO Cluster Summary</div>} />
+        <Route path="/" element={<Cluster />} />
         <Route
           path="/namespace/:namespaceId"
```
[the remark this answers is not in the document]
Comment (bbehnke) [reply ui/src/App.tsx:100]: @veds-g The routes are changing in v1. They are documented in the shared doc we have. 

I will checkout running it on 8443.

[the remark this answers is not in the document]
Comment (bbehnke) [reply ui/src/App.tsx]: @veds-g Plural route names are confirmed now so updated that. Also fixed the 404 issue. TY for catching that.

**Comment by veds-g** [comment]
looks good, one minor thing, I see a scrollbar for namespace and pipeline view for Breadcrumbs. can you check that?
image

**Comment by bbehnke** [comment]
> looks good, one minor thing, I see a scrollbar for namespace and pipeline view for Breadcrumbs. can you check that? image

@veds-g Updated. Good catch!

**Review by veds-g — APPROVED** [review]

**Pull request merged**